backend/src/main/java/com/woowacourse/levellog/prequestion/dto/PreQuestionWriteDto.java

```diff
@@ -0,0 +1,29 @@
+package com.woowacourse.levellog.prequestion.dto;
+
+import com.woowacourse.levellog.levellog.domain.Levellog;
+import com.woowacourse.levellog.member.domain.Member;
+import com.woowacourse.levellog.prequestion.domain.PreQuestion;
+import javax.validation.constraints.NotBlank;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@EqualsAndHashCode
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class PreQuestionWriteDto {
+
+    @NotBlank
+    private String preQuestion;
```

Comment:
```suggestion
    private String content;
```
로 변경하고 프론트에게 알려줬으면 좋겠어요 😊 
하는김에 InterviewQuestionWrite 에서도 content로 바꿔주세요.